lib/susedistribution.pm

```diff
@@ -430,14 +432,17 @@ sub activate_console {
     elsif ($type eq 'virtio-terminal') {
         serial_terminal::login($user);
     }
-    elsif ($type eq 'ssh' || $console eq 'svirt') {
+    elsif ($type eq 'ssh') {
         $user ||= 'root';
         assert_screen 'password-prompt';
```

Comment:
can you make that handle_password_prompt then too? I didn't know about the existance of this